docs/deployment.md

```diff
@@ -0,0 +1,75 @@
+
+
+## 3. Deploy to staging
+
+1. Load the [Find Postgraduate Teacher Training release pipeline](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_release?_a=releases&view=mine&definitionId=36)  in Azure DevOps.
+2. Click the blue `:rocket:` "Create Release" button  at the top right of the page which will open "Create a new release" sub-menu. 
+3. Release is made up of several stages i.e. backend, frontend, integration test. Deselect the stages that you wish to run manually by changing its trigger from automated to manual (optional).
+4. Under Artifacts section, master branch build artifacts should be already selected from the drop down list by default. Should you wish to change this select the version for the artifact sources for `manage-courses-backend`  and `manage-courses-frontend` drop down list.
+5. Specify release description (optional).
+6. Click the Create button to create the deployment.
+7. Lastly, approval your deployment by clicking on Appove button. Find development team members have sufficient rights to approve deployments. 
+
+## 4. Test on staging
+
+Integration test stage is automatically triggered after frontend deployment. 
+Take neccessary actions to test what you've just deployed. Be sure to keep an eye on Sentry for any incoming issues.
+[Sentry - Frontend](https://sentry.io/organizations/dfe-bat/issues/?project=1407453)
+[Sentry - Backend](https://sentry.io/organizations/dfe-bat/issues/?project=1377944)
+
+## 5. Deploy to production
+
+1. Load the release that triggered staging deployment [Find Postgraduate Teacher Training release pipeline](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_release?_a=releases&view=mine&definitionId=36) in Azure DevOps.
+2. Production deployment is configured to auto-trigger upon successful deployment of staging stage but requires a pre-deploymental approval.
+3. Click on the blue "Pending Approval" (clock) icon to approve/reject the release. Specify the comments and approve the release to trigger deployment or reject to terminate the release.
+
+## 6. Test on production
+
+Wait until the deploy finishes and, if necessary, test on production.
+Validate the production environment before declaring the deploy as complete.
+
+## 7. Move deploy cards to done
+
+Tell your team mates that their work has gone out, and move over all of the cards in "To deploy" to done on the [Find Team board](https://trello.com/b/fXA6ioZN/team-board-find-team).
+
+## Rolling back
+
+*Note that this advice does not apply if you are deploying changes to the Azure
+infrastructure. For rollback to the changes made to the infrasructure, the only way to roll
+back is to run a full redeploy.*
+
+Because we operate blue/green deployments, the previous version of the app is
+always available in the staging slot. To roll back to it, follow these
+instructions.
+
+1. [Obtain elevated permissions using Azure PIM](pim-guide.md)
```

Comment:
Does this exist yet? If not, can we make it clear it's not there yet?

Reply:
pim-guide should be there now